README.md

```diff
@@ -149,24 +128,52 @@ should be used that bootstraps the server. It installs Ansible's dependencies,
 and sets up a non-root user for Ansible to use. A playbook should be applied to
 a host by means of a wrapper script around `ansible-playbook`, that posts
 progress notifications to the committee's Slack team, among a few other things.
-To bootstrap the new server, the script should be run as follows:
-`$ ansible/scripts/run-playbook.sh <ENVIRONMENT>
-ansible/bootstrap-new-host.yml`, where `<ENVIRONMENT>` should be substituted by
-either `production` or `staging`.
 
 After the bootstrapping, the main playbook can be run to completely set up the
 server. The main playbook can be applied in the same way as the bootstrap
 playbook.
 
 When this has successfully finished, a server exists that matches one of the
-environments. When migrating the production server, a few more tasks should be
-performed, which is explained in detail in [this guide][deployment-new-production].
+environments.
+
+### Step-by-step guide
+These are the steps to follow to set up a new development or production server.
+Some of the steps require you to specify which of the two you are setting up.
+
+If you want to migrate from an existing server, a few additional tasks should be
+performed, which are explained in detail in [this guide][deployment-new-production].
+
+##### On Digital Ocean:
+1. Create a droplet named either `dev.svsticky.nl` (staging) or `svsticky.nl` (production).
+1. Assign a floating IP to the new droplet.
```

Comment:
Can you add the reason? (Because the floating IP's are already in DNS, which avoids DNS cache problems)